src/styles/style.css

```diff
@@ -0,0 +1,269 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  max-width: 707px;
+}
+
+.calendar__day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: #eee;
+  height: 100px;
+  width: 100px;
+  border: 1px solid black;
+}
+
+.calendar__day:hover {
+  cursor: pointer;
+  background-color: #ffbfcb;
+  transform: translateY(-20px);
+  animation-name: transform;
+  animation-duration: 500ms;
```

Comment:
The `animation-name` should be defined within the `@keyframes` rule, and you should define keyframes for the transform animation. Currently, the `transform` property is applied instantly because there are no keyframes defined for the animation named 'transform'.